project/docs/conf.py.j2

```diff
@@ -123,6 +125,8 @@ exclude_patterns = [
 
 autosummary_generate = False
 
+suppress_warnings = ["myst.header"]
```

Comment:
question(blocking): Should we really ignore this? I'm assuming you're including it because of [this issue](https://github.com/salt-extensions/saltext-freezer/commit/48880420e3564e04d7007dcc501d8451daeeae73), which makes me think something is wrong with the changelog generation. I'll look into it.

Reply:
Mh, when I tag a release for `pushover`, I get the following (syntactically correct) output:

```md
# Changelog

## 1.0.0 (2024-01-14)

No significant changes.
```

Where does the (invalid) version header in `saltext-freezer` come from then?

```md
# Changelog

# Saltext.Freezer 1.0.0 (2024-01-14) <--- How is this generated?

### Added

- Initial version of Freezer Extension for Salt. This release tracks the functionality in the core Salt code base as of version 3007.0
```

Reply:
Dunno... I ran into it when I was updating saltext-azurerm, which has been around for a while. I'll have to see if it's a legacy problem that I can run around and fix. Stand by.

Reply:
Looks like a legacy problem. Rolled back this change.